src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+let clicks = false;
+
+function createNotification(message) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.textContent = message;
+
+  if (message.includes('resolved')) {
+    div.classList.add('success');
+  }
+
+  if (message.includes('rejected')) {
+    div.classList.add('error');
+  }
+
+  document.body.appendChild(div);
+}
+
+const p1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    clicks = true;
+    resolve('First promise was resolved');
+  });
```

Comment:
The event listener for 'click' is added to the entire document, which might not be the most efficient approach if there are many elements on the page. Consider adding the event listener to a more specific element if possible.